includes/Abstracts/DokanRESTController.php

```diff
@@ -149,6 +157,23 @@ public function update_item( $request ) {
             }
 
             $object->save();
+
+            $author_id = dokan_get_current_user_id();
```

Comment:
Bhai, this portion of the code (From lines 161 to 175) has been used already in lines 105 to 119. So, please consider creating a separate function and using it in both `create_item` and `update_item` methods to avoid repetition.

Reply:
Refactor done. 

Thank you.